runtime-config/runtime.yml

```diff
@@ -49,6 +49,8 @@ addons:
 - include:
     stemcell:
     - os: ubuntu-bionic
+    deployments:
```

Comment:
So we're ensuring that the domain-broker gets the runtime config even if it's not on bionic?

Reply:
Yeah, main reason is this broker and its deployment is deprecated so trying to avoid making changes on that repo itself.